aksetup_helper.py

```diff
@@ -533,10 +533,10 @@ class BoostLibraries(Libraries):
     def __init__(self, lib_base_name, default_lib_name=None):
```

Comment:
Please keep this PR to a single topic (and file a separate PR for installation-related changes, if desired).

Reply:
Ok, removing changes